README.md

```diff
@@ -2,6 +2,13 @@
 
 The goal of *starfish* is to **prototype** a reference pipeline for the analysis of image-based transcriptomics data that works across technologies. This is a **work in progress** and will be developed in the open.
 
+We are currently in **pre-alpha**, finishing proof of concept pipelines for each of the spaceTx contributors that leverage starfish's shared object model.
+At this time starfish is mature enough to support computational developers interested in adapting other assays to starfish's object model.
+However, please note that these endeavors, while strongly encouraged, will likely come with significant computational time investment.
```

Comment:
computatonal time investment? What does that mean? I think we just nuke this sentence?